builders/integrations/wallets/rainbowkit.md

```diff
@@ -0,0 +1,328 @@
+
+--8<-- 'code/builders/integrations/wallets/rainbowkit/terminal/cli-quickstart.md'
+
+You can now navigate to the project directory, start the development server, and navigate to [http://localhost:3000](http://localhost:3000){target=\_blank} to view your project locally:
+
+=== "npm"
+
+    ```bash
+    cd <project-name>
+    npm run dev 
+    ```
+
+=== "pnpm"
+
+    ```bash
+    cd <project-name>
+    pnpm run dev
+    ```
+
+=== "yarn"
+
+    ```bash
+    cd <project-name>
+    yarn dev
+    ```
+
+Your starting screen should look like this:
+
+![Scaffolded RainbowKit project landing page](/images/builders/integrations/wallets/rainbowkit/rainbowkit-1.webp)
+
+Open the project in your code editor and take a look at the directory and file structure, making note of the `wagmi.ts` file. This file is where you can customize which chains to include in the list of networks users can connect to through your dApp. 
+
+Moonbeam, Moonriver, and Moonbase Alpha are not on the list of default supported networks. You can customize your dApp's supported networks in the `wagmi.ts` file by updating the chain entrypoints imported from `wagmi/chains` and passed to the `chains` property when `config` is defined. Wagmi uses [chain definitions](https://wagmi.sh/core/api/chains#available-chains) established by viem, primarily the chain name. The `wagmi/chains` names for Moonbeam networks are as follows: 
+
+=== "Moonbeam"
+
+    ```js
+    `moonbeam`
+    ```
+
+=== "Moonriver"
+
+    ```js
+    `moonriver`
+    ```
+
+=== "Moonbase Alpha"
+
+    ```js
+    `moonbaseAlpha`
+    ```
+
+To add support for Moonbeam networks, update `wagmi.ts` as follows. You will learn how to generate the `projectId` value in the next section. 
+
+```js title="src/wagmi.ts"
+
+--8<-- 'code/builders/integrations/wallets/rainbowkit/wagmi.ts'
+
+```
+## Manual Setup 
+
+If you want to add RainbowKit to an existing React application, you can complete a manual setup. The following sections will guide you through using the manual setup to install and import needed dependencies, configure chain connections to support Moonbeam networks, and make RainbowKit functionality available to users of your dApp. You will also learn how to specify which chain the **Connect Wallet** button should connect to by default and how to customize the RainbowKit theme to fit your project.
+
+### Checking Prerequisites {: #checking-prerequisites }
+
+The following guide assumes you have:
+
+- An existing dApp built with [React](https://react.dev/){target=\_blank} and you want to use the manual setup to connect to a mobile wallet via RainbowKit 
+
+    - This example uses [Next.js](https://nextjs.org/docs){target=\_blank} with Typescript and the App Router. You can find examples for additional frameworks in the [RainbowKit examples repository](https://github.com/rainbow-me/rainbowkit/tree/main/examples){target=\_blank}
```

Comment:
I'm a bit conflicted about the wording of these two bullet points. I read "This guide assumes you have an existing dApp built with React" and was like, ok great. Then I read that the example uses Next.js and App Router, fine. I assumed that adding those pieces would be part of the tutorial as we went, but they weren't. So I just feel like this needs to be more concise and like "You need a React project with Next.js, TypeScript, and the App Router"

Reply:
See if this is more clear now. I'm trying to convey they can use their preferred framework but, if they want the code example to work they should set it up a certain way.

Reply:
Yeah, that looks great! Thank you!